### DISCUSSION TITLE
再接続通知を削除したい

### DISCUSSION THREAD

**Discussion by hakatashi** [tsg-ut/slackbot#566] [投票所]
このトピックは GitHub Discussions を用いた投票のテストです。

以下は @ishitatsuyuki による提案文です (#526 より引用)。

> 再接続通知がデバッグに有用であることが少ない一方で、その投稿の頻度は低くありません。
>
> sandboxでのノイズを減らし、コミュニケーションの場としての役割を強化するため、再接続通知を削除します。

この提案に賛成の場合は :+1: を、反対の場合は :-1: でこのメッセージにリアクションしてください。

投票期限はこのメッセージがpostされてから1ヶ月以内とし、原則として賛成票と反対票の多い方を採用します。

補足意見がある場合はこのトピックにメッセージで返信してください。

**Comment by settyan117** [comment]
再接続の頻度が会話の邪魔になったと感じたことがありません

**Reply by naan4UGen** [reply]
sandboxをノイズを含んだ会話の場としてどの程度許容するかという話だと思っていて(#565 も同様)、会話の邪魔であると思う閾値がどのようであるかを判定するものとしてリアクションの統計が取られていそう(当然リアクションでは0/1しか取っていないという点でこのpostにも意味はあるんですが)

**Comment by hideo54** [comment]
個人的には「再接続されました」を見ることでサーバーの余裕度を知ることができ有用だと感じています。特にemodiやロボットバトルなどの重いbotを呼んだ時は、「再接続しました」を見てレスポンスの遅さに納得できることが多いです。

というわけで反対票を入れていますが、実際消えたところでそこまで困らないだろうなとも思っています。

**Reply by hideo54** [reply]
「最近はなぜか多いな」と感じており、削除提案に少し同情できるところですが、そういうふうに状態をなんとなく知れるところも良いところの1つと感じています。

**Comment by hakatashi** [comment]
本件は 賛成5:反対4 で可決されました。

**Comment by hakatashi** [comment]
#526 にて変更反映済みです
